collections/yen-d/meta.json

```
@@ -0,0 +1,10 @@
+{
+"name": "yen-D #01",
+"inscription_icon": "35881ac9a346c6991e139aae39c4b26961558a04408bc5abbc4c2e4261afa25ci0",
+"supply": "",
```

collection のアイテム数を入れた方がいいです。
```suggestion
"supply": "100",
```